src/components/Calculator.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import '../index.css';
+
+// Create a Class Component Calculator.
+class Calculator extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container justify-content-center">
+        <div className="row">
+          <div className="col-12">
+            <h1 className="display-5 fw-bolder text-center">CALCULATOR</h1>
+            <hr />
+          </div>
+        </div>
+        <div className="row justify-content-center">
+          <div className="col-md-8">
+            <div className="card mb-3">
+              <div className="card-body bg">
+                <div className="col-md-12 display num">0</div>
+                <div className="row d-flex justify-content-between mt-2">
+                  <div className="col-3">
+                    <button
+                      className="btn btn-light shadow p-4 fs-4"
+                      type="button"
+                    >
+                      AC
+                    </button>
+                  </div>
+                  <div className="col-3">
+                    <button
+                      className="btn btn-light shadow p-4 fs-4"
+                      type="button"
+                    >
+                      +/-
```

Comment:
- [ ] [OPTIONAL] One thing React excels in is components and their reusability. Here, I see an opportunity to create a `<Button />` component and reuse it while passing in the props you need 😎